packages/server/modules/core/rest/upload.js

```diff
@@ -74,13 +79,20 @@ module.exports = (app) => {
             `File upload of the multipart form has reached an end of file (EOF) boundary. The mimetype of the file is '${mimeType}'.`
           )
           if (requestDropped) return
-          const t0 = Date.now()
+          const objectBatchFileEndTime = Date.now()
           let objs = []
 
           const gzippedBuffer = Buffer.concat(buffer)
           if (gzippedBuffer.length > MAX_FILE_SIZE) {
             req.log.error(
-              `Upload error: Batch size too large (${gzippedBuffer.length} > ${MAX_FILE_SIZE})`
+              {
+                bufferLengthMb: toMegabytesTo1DecimalPlace(gzippedBuffer.length),
```

Comment:
There's a lot of duplication in these error info objects, I'd move those out to a function

```
const getErrorInfo = () => ({
  bufferLengthMb: gunzippedBufferMegabyteSize,
                maxFileSizeMb: toMegabytesTo1DecimalPlace(MAX_FILE_SIZE),
                elapsedTimeMs: Date.now() - start,
                objectBatchElapsedTimeMs: Date.now() - objectBatchFileEndTime,
                totalProcessed,
...
})
```